src/cmakeFindModules/AliceVisionConfig.cmake.in

```diff
@@ -1,11 +1,3 @@
-
-# Copyright (c) 2015 Pierre MOULON.
```

Comment:
A copyright was there!
```
# Copyright (c) 2016 AliceVision contributors.
# Copyright (c) 2015 Pierre MOULON.
```

or 
```
# Copyright (c) 2016 AliceVision contributors.
# Copyright (c) 2015 openMVG contributors.
```

Reply:
As a general rule, we avoid putting copyrights and licenses into configuration and build files. That would make things unclear if we make an exception for one file.
Is that acceptable for you if we remove this one?

Reply:
You can remove this one.

Reply:
Thanks